content/en-us/reference/engine/classes/SoundService.yaml

```diff
@@ -86,8 +86,29 @@ properties:
     capabilities: []
     writeCapabilities: []
   - name: SoundService.DefaultListenerLocation
-    summary: ''
-    description: ''
+    summary: |
+      Determines where (if anywhere) to place an `Class.AudioListener` by default.
+    description: |
+      Determines where to place an `Class.AudioListener` by default.
+      If `Class.SoundService.DefaultListenerLocation` is `None`, then no `Class.AudioListener` will be created by default, but `Class.AudioListeners` can be created separately by scripts.
+      If `Class.SoundService.DefaultListenerLocation` is `Camera`, then
+      - an `Class.AudioListener` will be created and parented-to `Class.Workspace.CurrentCamera`
+      - an `Class.AudioDeviceOutput` will be created and parented-to `Class.SoundService`
+      - a `Class.Wire` will be created and parented-to the previously-created `Class.AudioListener`
+      - `Class.Wire.SourceInstance` will be set to the previously-created `Class.AudioListener`, and `Class.Wire.TargetInstance` will be set to the previously-created `Class.AudioDeviceOutput`
+      So, the world will be heard from the perspective (postition and orientation) of your camera.
+
+      If `Class.SoundService.DefaultListenerLocation` is `Character`, then
+      - an `Class.Attachment` will be created and parented-to your `Class.Players.LocalPlayer`'s `Class.Player.Character`'s `Class.Model.PrimaryPart`
+      - an `Class.AudioListener` will be created and parented-to the `Class.Attachment`
+      - an `Class.AudioDeviceOutput` will be created and parented-to `Class.SoundService`
+      - a `Class.Wire` will be created and parented-to the previously-created `Class.AudioListener`
+      - `Class.Wire.SourceInstance` will be set to the previously-created `Class.AudioListener`, and `Class.Wire.TargetInstance` will be set to the previously-created `Class.AudioDeviceOutput`
+      - the previously-created `Class.Attachment` will be updated once-per-frame to face the same direction as `Class.Workspace.CurrentCamera`
```

Comment:
We could move this line to be indented under the earlier line about the Attachment